generators/createDisplayUnitSet/templates/shared/.sharedrc

```diff
@@ -50,14 +40,13 @@
     "css": "./css/style.css",
     "defaultFonts": [
       "OpenSans",
-      "OpenSansBold",
-      "OpenSansMedium"
+      "OpenSansBold"
     ],
     "defaultFontUrl": "./css/fonts.css",
     "clickTag": "https://www.google.com",
-    "text": "Welcome to this Banner!",
+    "title": "Welcome to this Banner!",
```

Comment:
"copy": "Welcome to this Banner!",

Reply:
Maybe we can change the variable name to 'copy' but I think this is ok as well